app/src/main/kotlin/com/wire/android/ui/common/LinkifyText.kt

```diff
@@ -215,19 +219,31 @@ private fun ClickableText(
     onLongClick: (() -> Unit)? = null,
 ) {
     val layoutResult = remember { mutableStateOf<TextLayoutResult?>(null) }
+    val tapped = remember { mutableStateOf(false) }
+    val interactionSource = remember { MutableInteractionSource() }
+
     val pressIndicator = Modifier.pointerInput(Unit) {
         detectTapGestures(
             onTap = { pos ->
                 layoutResult.value?.let { layoutResult ->
                     onClick(layoutResult.getOffsetForPosition(pos))
                 }
             },
+            onPress = { offset ->
+                tapped.value = true
+                val press = PressInteraction.Press(offset)
+                interactionSource.emit(press)
+                interactionSource.emit(PressInteraction.Release(press))
+                tapped.value = false
+            },
```

Comment:
What does it do exactly and why is it needed after merging messages from the same sender? Isn't it handled automatically? 😄 
It can potentially handle taps when it shouldn't:
> After the initial [onPress](https://developer.android.com/reference/kotlin/androidx/compose/foundation/gestures/package-summary#(androidx.compose.ui.input.pointer.PointerInputScope).detectTapGestures(kotlin.Function1,kotlin.Function1,kotlin.coroutines.SuspendFunction2,kotlin.Function1)), if the pointer moves out of the input area, the position change is consumed, or another gesture consumes the down or up events, the gestures are considered canceled.

Reply:
It seems that this workaround it's not needed anymore.